tests/expected/configured/pmt2_0_0_0.png.txt

```diff
@@ -1 +1 @@
-tests/output/configured/pmt2_0_0_0.png: RIFF (little-endian) data, Web/P image
+tests/output/configured/pmt2_0_0_0.png: RIFF (little-endian) data, Web/P image, with alpha, 511+1x511+1
```

Comment:
my local environment and CI disagree if this file have alpha or not.

No clue why, likely a newer version of this detector.
In any case an unrelated bug.

```suggestion
tests/output/configured/pmt2_0_0_0.png: RIFF (little-endian) data, Web/P image
```